src/_versions.ts

```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.2.0',
```

Comment:
MINOR version should be updated in package.json as this introduces new functionality.

https://semver.org/

Reply:
@CFIALeronB don't see a reply to this one.

Reply:
@rngadam done!